hotsos/defs/tests/scenarios/openstack/openstack_charm_conflicts_fail.yaml

```diff
@@ -0,0 +1,14 @@
+target-name: openstack_charm_conflicts.yaml
+mock:
+  patch:
+    hotsos.core.plugins.juju.JujuChecksBase.charms:
+      kwargs:
+        new:
+          neutron-api: foo
+          neutron-gateway: foo
+          nova-compute: foo
+raised-issues:
+  OpenstackWarning: >-
+    This host is running both neutron-api and neutron-gateway charms which
+    is not supported and will lead to unexpected behaviour such as conflicts
+    when more than one charm writes neutron.conf at once.
```

Comment:
nit: there isn't a specific test for nova

Reply:
fair point, i could trigger both in the same test so ill update